src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/ReportFilters/ReportFilters.styles.ts

```diff
@@ -21,3 +21,14 @@ export const StyledMoreFilters = styled(Button)`
   height: 5.5rem;
   margin-left: ${theme.spacing(1.2)};
 `;
+
+export const StyledCheckboxTitle = styled(StyledBodyMedium)`
+  display: flex;
+  align-items: center;
+
+  svg {
+    height: 1.6rem;
+    margin-left: ${theme.spacing(0.4)};
+    margin-top: ${theme.spacing(0.4)};
```

Comment:
Nit: the misalignment can be better corrected by the display mode than by adding magic margin values:
```suggestion
    display: block;
```

Reply:
The `margin-left` was because the icon is slightly too far from the label, but overall I like your suggestion better, thanks!